src/yuzu/configuration/configure_input_player.cpp

```diff
@@ -119,6 +119,22 @@ QString GetButtonName(Common::Input::ButtonNames button_name) {
     }
 }
 
+QString GetDirectionName(const std::string& direction) {
```

Comment:
```suggestion
QString GetDirectionName(std::string_view direction) {
```

It's preferable to use a `string_view` here